aten/src/ATen/native/native_functions.yaml

```diff
@@ -14639,6 +14639,21 @@
     NestedTensorCUDA: NestedTensor_to_padded_tensor_cuda
   autogen: to_padded_tensor.out
 
+- func: _fbgemm_jagged_to_padded_dense_forward(Tensor values, Tensor[] offsets, SymInt[] max_lengths, float padding_value=0.0) -> Tensor
+  variants: function
+  dispatch:
+    CUDA: _fbgemm_jagged_to_padded_dense_forward
+
+- func: _fbgemm_jagged_to_padded_dense_backward(Tensor grad_output, Tensor[] offsets, SymInt total_L) -> Tensor
```

Comment:
There's a minor difference in whether `total_L` is required or not. I think in general, it's better for it not to be required because getting that value from `offsets[-1]` incurs a GPU sync.

So I think agree with you that we just need `jagged_to_padded_dense_forward` and `dense_to_jagged_forward`. I'll try removing the backward op.

Reply:
Removed the backward op